docs/config-fcos-v1_6-exp.md

```diff
@@ -209,8 +209,9 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
   * **_should_exist_** (list of strings): the list of kernel arguments that should exist.
   * **_should_not_exist_** (list of strings): the list of kernel arguments that should not exist.
 * **_boot_device_** (object): describes the desired boot device configuration. At least one of `luks` or `mirror` must be specified.
-  * **_layout_** (string): the disk layout of the target OS image. Supported values are `aarch64`, `ppc64le`, and `x86_64`. Defaults to `x86_64`.
+  * **_layout_** (string): the disk layout of the target OS image. Supported values are `aarch64`, `ppc64le`, `s390x-eckd`, `s390x-virt`, `s390x-zfcp` and `x86_64`. Defaults to `x86_64`.
   * **_luks_** (object): describes the clevis configuration for encrypting the root filesystem.
+    * **device-s390x** (string): describes device specific to s390x `dasd[a-z]` or `sd[a-z]`.
```

Comment:
We should follow the same semantics that are pre-existing, i.e. s390x_device

Reply:
and I am a bit confused; looking at the linked issue
```

variant: openshift
version: 4.13.0
metadata:
  name: worker-storage
  labels:
    machineconfiguration.openshift.io/role: worker
boot_device:
  layout: s390x-zfcp
  luks:
    device: /dev/sd[a-z]
    tang: 
        - url: http://tang1.example.com:7500
          thumbprint: jwGN5tRFK-kF6pIX89ssF3khxxX
          
        
```

None of the shared configs show any new fields in the boot device? 

So why are we adding this new field?

Reply:
1. We should follow the same semantics that are pre-existing, i.e. s390x_device
> Will use `s390x_device`

2. None of the shared configs show any new fields in the boot device?
3. So why are we adding this new field?
> Ans for 2 and 3.  in the shared config I was giving an overview of how the sugar should be and what output it provides. From the comments  starts https://github.com/coreos/butane/issues/453#issuecomment-1651023797  Benjamin suggested to add a device field under `boot_device.luks.device` so that it does not conflict with the semantic device  list in mirror. The new device field specifically for s390x layouts.

Reply:
Added `luks.s390x-device` as suggested.